depmapomics/qc/config.py

```diff
@@ -25,7 +25,8 @@
                     '21q3': {'internal': {'name': 'internal-21q3-fe4c', 'version': 12},
                              'ibm': {'name': 'ibm-21q3-179f', 'version': 8},
                              'dmc': {'name': 'dmc-21q3-482c', 'version': 7},
-                             'public': {'name': 'public-21q3-bf1e', 'version': 7}}
+                             'public': {'name': 'public-21q3-bf1e', 'version': 7}},
+                    '21q1': {'internal': {'name': 'internal-21q1-4fc4', 'version': 40}}
```

Comment:
Why are we comparing the current release with the one two quarters ago?

Reply:
If you look at lines 32 and 33 the comparison is between 21q2 and 21q3. But sometimes we realize that something unexpectedly changed in a quarter and was reverted again, so comparing to the quarter before that is sometimes helpful for some of these cases.